src/Player/Player.cs

```diff
@@ -1,11 +1,23 @@
 public class Player
 {
-  public string playerName = "Stepper";
-
-  public int playerHealth = 10;
-  public int playerStepLength = 1;
+  string playerName = "Stepper";
+  int playerHealth = 10;
+  int playerStepLength = 1;
+  int walkedDistance = 0;
```

Comment:
We should use concise naming for the class. The health property for the player can be called `health` instead of `playerHealth` as she is already a part of a class called `Player`

Reply:
```suggestion
  string name = "Stepper";
  int health = 10;
  int stepFactor = 1;
  int walkedSteps = 0;
```